### DISCUSSION TITLE
Option on converter front end to convert to 476?

### DISCUSSION THREAD

**Discussion by TinWiz** [ParadoxGameConverters/ImperatorToCK3#232] [Ideas]
Could we add the option in the converter to convert to a 476 starting date? This might have to wait until the parameters for converting development are set. I'm not sure if this would cause issues because of the significantly longer CK3 campaign it would lead to.

**Comment by IhateTrains** [comment]
It's possible if we use WtWSMS or Fallen Eagle as a base.
